holidays/countries/turkmenistan.py

```
@@ -0,0 +1,69 @@
+        self._add_eid_al_fitr_day_two("Eid al-Fitr")
+        self._add_eid_al_fitr_day_three("Eid al-Fitr")
+        
+        self._add_eid_al_adha_day("Eid al-Adha")
+        self._add_eid_al_adha_day_two("Eid al-Adha")
+        self._add_eid_al_adha_day_three("Eid al-Adha")
+
+class TM(Turkmenistan):
+    pass
+
+class TKM(Turkmenistan):
+    pass
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)